apps/src/weblab/brambleHost.js

```diff
@@ -497,17 +497,16 @@ function load(Bramble) {
       // Remove leading project root path
       var cleanedPath = path.replace(removeProjectRootRegex, '');
       // If a 'change' operation is already queued for the same file, return
-      for (var i = 0; i < _recentBrambleChanges.length; i++) {
-        let change = _recentBrambleChanges[i];
-        if (change.operation === 'change' && change.file === cleanedPath) {
-          return;
```

Comment:
The problem was this early return, which caused us to skip the `onProjectChangedCallback_` on some change events.  @cpirich suggested we make this step only skip the `_recentBrambleChanges.push` to avoid duplicating entries in that array, but still always hit the callback.  In testing this appears to solve the issue.